src/patterns/ec2-app.ts

```diff
@@ -180,17 +180,6 @@ export class GuEc2App {
       vpc,
       internetFacing: true,
       vpcSubnets: { subnets: GuVpc.subnetsfromParameter(scope, { type: SubnetType.PUBLIC, app }) },
-      securityGroup:
```

Comment:
I think defining a custom security group here is unnecessary. By default, [AWS will configure a security group](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-elasticloadbalancingv2.ApplicationLoadBalancer.html#securitygroup) with rules which allow the load balancer to talk to instances (and vice versa) over the correct port only.